MedicationScheduler/MedicationScheduler/urls.py

```diff
@@ -13,9 +13,16 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
+
 from django.contrib import admin
 from django.urls import path
+from AuthApp.views import registration_view, login_view, logout_view
+from MedSchedulerApp.views import home_view
 
 urlpatterns = [
+    path('', home_view, name='home'),
```

Comment:
I think in Django we can separate the URLs based on the app then include them into the main `urls.py` file. This is an example from the RMPPlus website:
```
urlpatterns = [
    path('admin/', admin.site.urls),
    path('', include('rmp_bd_app.urls')),
]
```
I think this will help with readability in the future when we are using the URLs in the html pages.

Reply:
Done